keyboards/crkbd/keymaps/shinze/rules.mk

```diff
@@ -0,0 +1,31 @@
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```

Comment:
These can be condensed down into just:

```make
RGBLIGHT_ENABLE = yes
```

as the rest of them are unchanged from the keyboard-level `rules.mk`.